docs/podman-varlink.1.md

```diff
@@ -0,0 +1,37 @@
+% podman(1) podman-varlink - Waits on a container
+% Brent Baude
+# podman-varlink "1" "April 2018" "podman"
+
+## NAME
+podman varlink - Runs the varlink backend interface
+
+## SYNOPSIS
+**podman varlink**
+[**--help**|**-h**]
+VARLINK_URI
+
+## DESCRIPTION
+Waits on one or more containers to stop.  The container can be referred to by its
```

Comment:
This paragraph makes no sense... Did you reuse `podman stop` but s/stop/varlink?

Reply:
lol, this is a copy/paste leftover. fixed